.pipelines/onebranch/templates/template-buildrp-buildaro.yml

```diff
@@ -12,3 +12,11 @@ steps:
       mkdir -p $(ob_outputDirectory) 
       cp aro $(ob_outputDirectory)/aro
     workingDirectory: $(Build.SourcesDirectory)
+- task: Bash@3
```

Comment:
@mikeandescavage and concerns over this task being inserted here given the refactor you're doing in https://github.com/Azure/ARO-RP/pull/1991?

Reply:
No concerns here.  Once this is merged to master, we'll merge it into our working branch and integrate.